docker-entrypoint.sh

```diff
@@ -29,12 +54,16 @@ clear_redis() {
   echo "Redis cleanup complete"
 }
 
-# See if we need to do database setup before starting.
-if [[ $DATABASE_SETUP == "1" ]]; then
-  # Run our database setup
-  database_setup
+# See if we need to do database wipe and reinitialize before starting.
+if [[ $DANGER_DATABASE_WIPE_REINITIALIZE == "1" ]]; then
+  # Run database wipe and reinitialize
+  DANGER_DATABASE_WIPE_REINITIALIZE
```

Comment:
If this is the fn call on line 10, should this not be lower case?

Reply:
You are right.